src/app/components/NavBar/__snapshots__/index.spec.tsx.snap

```diff
@@ -1135,6 +1135,31 @@ exports[`content in browser matches snapshot for logged out 1`] = `
   margin: 0 auto;
 }
 
+.c1 {
+  overflow: visible;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -webkit-justify-content: space-between;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  height: 6rem;
+  max-width: 128rem;
+  margin: 0 auto;
+}
+
+.c2 {
+  display: block;
+  width: auto;
+  height: 3.5rem;
+}
```

Comment:
Something with the snapshots is duplicating styles, there's multiple copies of the same rules in this spec...